### DISCUSSION TITLE
账号验证问题

### DISCUSSION THREAD

**Discussion by flyice-juin** [XiaoMi/ha_xiaomi_home#26] [Ideas]
目前只支持homeassistant.local:8123的验证url，但是很多人都不用HAOS，麻烦可以支持其他的ha的url连接

**Comment by JimmyKmi** [comment]
解决方案：https://github.com/XiaoMi/ha_xiaomi_home/discussions/6

**Reply by flyice-juin** [reply]
我这样改了，但是不行，一直转圈
